contrib/lame/module.defs

```diff
@@ -18,3 +18,12 @@ LAME.EXTRACT.tarbase = lame
 ifneq (none,$(FFMPEG.GCC.g))
 	LAME.CONFIGURE.extra += --enable-debug
 endif
+
+ifeq (1-msys,$(BUILD.cross)-$(HOST.system))
```

Comment:
should be addressed now but cannot fully check now,
hopefully later this week...